src/main/java/cart/service/CartService.java

```diff
@@ -13,7 +16,29 @@
 public class CartService {
     private final CartRepository cartRepository;
 
+    @Transactional
     public List<Cart> getList() {
         return cartRepository.findAll();
     }
+
+    @Transactional
+    public List<Cart> getList(Member member) {
+        return cartRepository.findByMember(member);
+    }
```

Comment:
조회만 하는 경우에 readyOnly 옵션을 활용해 보시는 것도 추천드립니다 
```suggestion
    @Transactional(readOnly = true)
    public List<Cart> getList(Member member) {
        return cartRepository.findByMember(member);
    }
```

Reply:
좋은 정보 감사합니다👍